buidl/psbt.py

```diff
@@ -628,6 +628,11 @@ def serialize(self):
             result += psbt_out.serialize()
         return result
 
+    def remove_global_xpubs(self):
+        """Blank the `hd_pubs` and return the PSBT in base64 serialized format"""
+        self.hd_pubs = {}
+        return self.serialize_base64()
```

Comment:
I don't think it's strictly necessary to `return` anything in this method, but it does no harm.

Reply:
Haha, that was my thinking as well. Otherwise, the method barely does enough to justify its existence.